Common/Inc/Interchip.h

```diff
@@ -5,7 +5,12 @@
 #define INTERCHIP_H
 
 typedef struct {
-	uint16_t safetyLevel;
-	int16_t PWM[12];
-} Interchip_Packet;  
+    // todo: set safety level to uint8_t
+    uint16_t safetyLevel;
+
+    // to be converted to be int8_t
+    // do we need signed integers? It seems like we might?
+    int8_t PWM[12];
```

Comment:
Im assuming you've verified that this works? I originally had issues with this, if I remember correctly.